packages/app/background-jobs-common/src/background-job-processor/monitoring/BackgroundJobProcessorMonitor.ts

```diff
@@ -89,19 +90,33 @@ export class BackgroundJobProcessorMonitor<
   public jobStart(job: JobType, requestContext: RequestContext): void {
     const transactionName = `bg_job:${this.ownerName}:${this.queueId}`
     this.transactionObservabilityManager.start(transactionName, resolveJobId(job))
-    requestContext.logger.info({ origin: this.processorName }, `Started job ${job.name}`)
+    requestContext.logger.info(this.buildLogParams(job), `Started job ${job.name}`)
+  }
+
+  public jobTryError(job: JobType, error: unknown, requestContext: RequestContext): void {
+    requestContext.logger.error(this.buildLogParams(job, error), `${job.name} try failed`)
   }
 
   public jobEnd(job: JobType, requestContext: RequestContext): void {
     requestContext.logger.info(
       {
-        origin: this.processorName,
+        ...this.buildLogParams(job),
         isSuccess: job.progress === 100,
       },
       `Finished job ${job.name}`,
     )
     this.transactionObservabilityManager.stop(resolveJobId(job))
   }
+
+  private buildLogParams(job: JobType, error?: unknown) {
+    return {
+      origin: this.processorName,
+      jobId: resolveJobId(job),
```

Comment:
isn't jobId coming from the childLogger already?

Reply:
you are completely right! I will revisit what is on child logger and remove from here, sorry 🙏

Reply:
Done, thanks 🙏